dev/InfoBar/InfoBar.cpp

```diff
@@ -161,7 +168,8 @@ void InfoBar::UpdateVisibility(bool notify, bool force)
                     auto const notificationString = StringUtil::FormatString(
                         ResourceAccessor::GetLocalizedStringResource(SR_InfoBarOpenedNotification),
                         Title().data(),
-                        Message().data());
+                        Message().data(),
+                        ResourceAccessor::GetLocalizedStringResource(GetSeverityLevelResourceName(Severity())));
```

Comment:
I need help with this. The formatting string is: "%1!s! %2!s! %3!s!" however this does not format correctly. The call to get the localized severity returns a correct string (tested using variable and debugger) so I have no idea why this does still show up as "{Title} {Message}". @StephenLPeters @ranjeshj Do you have any idea how this could happen or what could cause this?

Reply:
I assume you have already tried looking in the codebase for other example usages? I see this one [here](https://github.com/microsoft/microsoft-ui-xaml/blob/cb19f34a15115f889ba1854d2ed006792e9d0982/dev/AnimatedIcon/AnimatedIcon.cpp#L237).

Reply:
The following code also fails, despite `warningstring` having text:

```
const auto warningString = ResourceAccessor::GetLocalizedStringResource(GetSeverityLevelResourceName(Severity()));
auto const notificationString = StringUtil::FormatString(
    L"%1!s!%2!s!%3!s!",
    Title().data(),
    Message().data(),
    warningString);

winrt::get_self<InfoBarAutomationPeer>(peer)->RaiseOpenedEvent(Severity(), notificationString);
```

Reply:
Was able to fix this issue, forgot to add `.data()` at the end of the string lookup. Sorry about any inconveniences caused.